### PR TITLE
fix(service-portal): empty state message for incoming deligations

### PR DESCRIPTION
**Pull request by Herdismaria**
## What

Fix wrong empty state message for incoming delegations.

## Why

To show clearer message to user.

## Screenshots / Gifs

Screenshot 2024-02-16 at 08 28 04

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/portals/shared-modules/delegations/src/lib/messages.ts [hunk 1]
```diff
@@ -6,10 +6,14 @@ export const m = defineMessages({
     defaultMessage: 'Veldu umboð',
     description: 'Choose delegation',
   },
-  noDelegations: {
-    id: 'sp.access-control-delegations:empty',
+  noOutgoingDelegations: {
+    id: 'sp.access-control-delegations-outgoing:empty',
```
Comment (GunnlaugurG) [inline libs/portals/shared-modules/delegations/src/lib/messages.ts:10]: We traditionally try to match the namespaces of the id for each translation file (with some unique exceptions) so the `-outgoing` and `-incoming` should be after the `:` (`'sp.access-control-delegations:empty-outgoing'`)

File: libs/portals/shared-modules/delegations/src/lib/messages.ts [hunk 2]
```diff
@@ -6,10 +6,14 @@ export const m = defineMessages({
...
     defaultMessage: 'Umboð sem þú hefur veitt öðrum munu birtast hér.',
   },
+  noIncomingDelegations: {
+    id: 'sp.access-control-delegations-incoming:empty',
```
Comment (GunnlaugurG) [inline libs/portals/shared-modules/delegations/src/lib/messages.ts:14]: Same as above 😄

**Review by GunnlaugurG — COMMENTED** [review]
Great first PR 🤌 small translation fix and dont forget to run "extract-string" to upload translations to contentfull

**Review by snaerseljan — APPROVED** [review]
Nice work! First PR of many! 😀
Just remember to update Contentful, i.e. yarn extract-strings. I don't remember if we did English translations there as well 🤔

**Pull request merged**